src/cms/content-staging-dashboard.md

```diff
@@ -3,32 +3,34 @@ ee_only: true
 title: Staging Dashboard
 ---
 
-The Staging Dashboard provides an overview of all active and upcoming campaigns. The format of the dashboard can be changed from a grid to a timeline. You can also use filters to find campaigns, customize the column layout, and save different views of the grid. To learn more about the workspace controls, see: [Admin Workspace]({% link stores/admin-workspace.md %}).
+The Staging Dashboard provides an overview of all active and upcoming campaigns. The format of the dashboard can be changed from a grid to a timeline. You can also use filters to find campaigns, customize the column layout, and save different views of the grid. For more information about the workspace controls, see [Admin Workspace]({% link stores/admin-workspace.md %}).
 
 ![]({% link images/images-ee/content-staging-grid-view.png %}){: .zoom}
-_Grid View_
+_Staging Dashboard in grid view_
 
-## To open the Staging Dashboard:
+## View the Staging Dashboard
 
-On the _Admin_ sidebar, go to  **Content** > _Content Staging_ > **Dashboard**.
+1. On the _Admin_ sidebar, go to  **Content** > _Content Staging_ > **Dashboard**.
 
-1. To change the format of the dashboard, set the **View As** control to `Timeline`.
+1. To change the format of the dashboard, set the **View As** control to `list`, `Grid`, or `Timeline`.
 
-    ![]({% link images/images-ee/content-staging-dashboard-timeline.png %}){: .zoom}
-    _Timeline View_
+   ![]({% link images/images-ee/content-staging-dashboard-timeline.png %}){: .zoom}
+   _Timeline view_
 
-    The slider in the lower-right corner can be used to adjust the view from one to four weeks. Each column represents one day.
+   The slider in the lower-right corner can be used to adjust the view from one to four weeks. Each column represents one day.
 
-1. Drag the slider to the `4w` position on the far right. The campaign that is scheduled for June 20 is now visible in the timeline.
+1. If the timeline is displayed, drag the slider to the `4w` position on the far right.
 
-    ![]({% link images/images-ee/content-staging-timeline-4-week-view.png %}){: .zoom}
-    _4-Week View_
+   In the following example, the campaign that is scheduled for June 20 is now visible in the timeline.
```

Comment:
maybe put the date in code?

Reply:
I actually am taking this line out because it really goes with the old screen cap and isn't really all that helpful.